lib/lighthouse_intent_to_file/tag_sentry.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module LighthouseIntentToFile
+  module TagSentry
```

Comment:
I think this can be removed, only other place Sentry is used is PensionBurial.  I recall being told that Sentry is being dropped in favor of Datadog